### DISCUSSION TITLE
ImageGlass startup speed

### DISCUSSION THREAD

**Discussion by Galaxoid-333** [d2phap/ImageGlass#1513] [Q&A]
Tell me the actual methods to speed up the launch of the application and the opening of images. ImageGlass is noticeably slower than other viewers. There is some reason for this.

**Comment by d2phap** [comment]
You can check the option:
`Start with OS to improve startup speed`

Please note that, if you uninstall ImageGlass, you need to deselect this option to remove ImageGlass from startup folder of Windows.

![image](https://user-images.githubusercontent.com/3154213/227075237-04233ea4-654f-4fee-ade0-7f88fb44e7eb.png)

**Reply by Galaxoid-333** [reply]
I'm talking about something else. The slow speed of opening images is noticeable even when scrolling the mouse. This is well understood when compared with other viewers. Changing the interpolation method has no effect.

**Reply by d2phap** [reply]
Because you asked about "the launch of the application" so my answer was the "Start with OS" option.
You can improve the loading speed by:
- Disable **Color management** option in **Image** tab which usually consumes lots of time.
- Or try ImageGlass 9.0 beta (with Color profile is disabled, too)

If both do not work for you, it's platform limitation (C# WinForms) and I have no other solution.

**Reply by Galaxoid-333** [reply]
Thanks for the answer. Unfortunately, these methods did not help. I haven't tried the beta version yet.
I have another question. For a second of waiting for the image to load, the mouse cursor changes to a waiting graphics (circle) and scroll bars are visible on the right and bottom. It should be?

**Comment by Galaxoid-333** [comment]
I installed version 7.6. It does not have a slow start and lags when moving to the next picture. Does version 8 have higher requirements? But in version 7.6 there are no image interpolation methods from 8, and the image is worse than in 8. Is this due to faster performance in 7?

Another guess. Could this be related to the caching of the following images?
